R/plot_carp.R

```diff
@@ -80,6 +80,11 @@
 #' carp_fit <- CARP(presidential_speech)
 #' plot(carp_fit, type='interactive')
 #' }
+
+if(getRversion() >= "2.15.1")  {
+  utils::globalVariables(c("V1", "V2", "Obs", "FirstV1", "FirstV2", "final_cluster", "ObsLabel", "FirstObsLabel"))
+}
+
```

Comment:
This is a pretty ugly hack. Did you look at the fix I recommended? (https://stackoverflow.com/questions/9439256/how-can-i-handle-r-cmd-check-no-visible-binding-for-global-variable-notes-when/57496617#57496617)?